assets/src/components/mapPage/routePropertiesCard.tsx

```diff
@@ -318,4 +314,33 @@ const RoutePropertiesCard = ({
   )
 }
 
+const RoutePropertiesCard = ({
+  routePatterns,
+  selectedRoutePatternId,
+  selectRoutePattern,
+  onClose,
+}: {
+  routePatterns: ByRoutePatternId<RoutePattern>
+  selectedRoutePatternId: RoutePatternId
+  selectRoutePattern: (routePattern: RoutePattern) => void
+  onClose?: () => void
+}) => {
+  const selectedRoutePattern = routePatterns[selectedRoutePatternId]
+  const route: Route | null = useRoute(selectedRoutePattern?.routeId)
+
+  if (!route || selectedRoutePattern === undefined) {
+    return <></>
+  }
+
+  return (
+    <RoutePropertiesCardRender
+      routePatterns={routePatterns}
+      route={route}
+      selectedRoutePattern={selectedRoutePattern}
+      selectRoutePattern={selectRoutePattern}
+      onClose={onClose}
+    />
+  )
+}
```

Comment:
suggestion(non-blocking): What do you think about this? I know it'd cause a diff (:/) in the rest of the application (and the `Render` suffix would need to be left off)

```suggestion
const RoutePropertiesCardFromPatternId = ({
  routePatterns,
  selectedRoutePatternId,
  selectRoutePattern,
  onClose,
}: {
  routePatterns: ByRoutePatternId<RoutePattern>
  selectedRoutePatternId: RoutePatternId
  selectRoutePattern: (routePattern: RoutePattern) => void
  onClose?: () => void
}) => {
  const selectedRoutePattern = routePatterns[selectedRoutePatternId]
  const route: Route | null = useRoute(selectedRoutePattern?.routeId)

  if (!route || selectedRoutePattern === undefined) {
    return <></>
  }

  return (
    <RoutePropertiesCardRender
      routePatterns={routePatterns}
      route={route}
      selectedRoutePattern={selectedRoutePattern}
      selectRoutePattern={selectRoutePattern}
      onClose={onClose}
    />
  )
}

RoutePropertiesCard.FromPatternId = RoutePropertiesCardFromPatternId
```

Reply:
Oh hey - it doesn't actually cause a diff with the rest of the app, because of 👇 

```ts
export default RoutePropertiesCardFromPatternId
```